src/delphi/train/training_old.py

```diff
@@ -16,6 +16,9 @@
 (If your cluster does not have Infiniband interconnect prepend NCCL_IB_DISABLE=1)
 """
 
+import torch._dynamo
+torch._dynamo.config.suppress_errors = True
```

Comment:
what is it?

Reply:
llama2.c allows to use just-in-time compiling (using TorchDynamo) to run inference on the compiled model in order to increase training speed, afaik. But for some reason this doesn't work on the servers that I am using to train the models, so I set "compile" to False and suppressed errors because there was some information being printed while training. Not using JIT compilation makes training a bit slower, but doesn't change anything about the model or training.